src/routes/Dashboard/components/PopUpCard/PopUpCard.scss

```diff
@@ -0,0 +1,22 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  overflow: hidden;
+}
+
+.mainSection {
+  padding: .5em;
```

Comment:
best practice to use rem for spacing.
Actually, you generally want to stay away from `em` unless you have a very good / specific reason to use it...such as some sort of font based thing.